cypress/support/Entities.js

```diff
@@ -6,6 +6,21 @@
 import * as modelPage from '../support/components/ModelPage'
 import * as entitiesGrid from './components/EntitiesGrid'
 import * as entityModal from '../support/components/EntityModal'
+import * as actionsGrid from './components/ActionsGrid'
+
+export const pretrainedEntityTypes = [
+  "datetimeV2",
+  "number",
+  "ordinal",
+  "percentage",
+  "temperature",
+  "dimension",
+  "money",
+  "age",
+  "url",
+  "email",
+  "phonenumber",
```

Comment:
Maybe you intended not to add them, but since you're editing this anyways I would remind of the new types `geographyV2, personName, and keyPhrase`.

Reply:
It was intentional because I wasn't ready to test this change, but I just now made and tested the change since it really is simple. I'll push that in with the rest of this code.